pre_requirement/terraform.auto.tfvars

```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-namespace     = "example"
-project       = "example"
-creator_email = "admin@example.co"
-owner_email   = "admin@example.co"
+namespace     = "wajahat"
```

Comment:
please remove wajahat and add example here

Reply:
Changed it to example